core/templates/components/state-editor/state-interaction-editor/state-interaction-editor.directive.ts

```diff
@@ -215,7 +218,19 @@ angular.module('oppia').directive('stateInteractionEditor', [
             );
           };
 
+          $scope.isInteractionDisabled = function() {
+            if (
+              StateInteractionIdService.displayed === 'EndExploration' &&
+              ContextService.isExplorationLinkedToStory()) {
```

Comment:
Simpler to just return this?

Reply:
Would it be possible to compute this on initialising this directive and use that value in the HTML?

Reply:
Done